extensions/reactive-messaging-http/deployment/src/main/java/io/quarkus/reactivemessaging/http/ReactiveHttpProcessor.java

```diff
@@ -0,0 +1,73 @@
+package io.quarkus.reactivemessaging.http;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.reactivemessaging.http.config.HttpStreamConfig;
+import io.quarkus.reactivemessaging.http.config.ReactiveHttpConfig;
+import io.quarkus.reactivemessaging.http.config.WebsocketStreamConfig;
+import io.quarkus.vertx.http.deployment.RouteBuildItem;
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ *         <br>
+ *         Date: 28/08/2019
+ */
```

Comment:
Same as usual, could you get rid of all the author tags? Better use git to track that. 

Thanks.

Reply:
I really, really, really like my author tags :(

Reply:
Again, this is project policy. We decided to not include them in our source code (at least, not more of them). AFAICS, there are still some in this patch.